share/wordlists/eff_large.wordlist

```diff
@@ -2006,8 +2006,8 @@ drizzly
 drone
 drool
 droop
-drop-down
-dropbox
+dropforge
+drop-in
```

Comment:
Should there be any dashes in the wordlist? I checked the file, there are only a few occurrences, so maybe worth replacing them?

Also, how is the dash ordered relative to other characters? Here you have `f` -> `-` -> `k`, which is rather arbitrary.